packages/components/src/components/data-table/table-row-info.jsx

```diff
@@ -13,6 +14,28 @@ const TableRowInfo = ({ replace, is_footer, cells, className }) => {
         }
     };
 
+    const CopyButton = ({ text_copy }) => {
+        const [icon, setIcon] = React.useState('IcClipboard');
+
+        const success = () => {
+            setIcon('IcCheckmarkCircle');
+            setTimeout(() => {
+                setIcon('IcClipboard');
+            }, 1000);
+        };
+
+        const copyText = event => {
+            navigator.clipboard.writeText(text_copy);
+            event.stopPropagation();
+            success();
+        };
+
+        return <Icon icon={icon} onClick={event => copyText(event)} />;
+    };
+    CopyButton.propTypes = {
+        text_copy: PropTypes.string,
+    };
+
```

Comment:
`CopyButton` component will be created on each render and I think it will lower the performance.
Can we move this component outside of `TableRowInfo` component?

Reply:
refactored, its not a component anymore